mcs/class/Mono.Posix/Mono.Unix.Native/Stdlib.cs

```diff
@@ -378,7 +380,11 @@ static XPrintfFunctions ()
 	//
 	public class Stdlib
 	{
+#if NETSTANDARD2_0
+		internal const string LIBC = "c";
```

Comment:
How will this assembly be used? What is the "larger" packaging context?

Specifically, will it be possible to use this ".NET Standard" assembly on desktop .NET? If so, that means that desktop .NET can no longer use the `Stdlib` class, whereas before it could be used there. That would be unfortunate.

If instead the ".NET Standard" profile assembly is only for .NET Core, and e.g. a bait-and-switch PCL will be used to use the "normal" assembly for desktop .NET, that might work...